src/py/_pyodide/_core_docs.py

```diff
@@ -441,6 +447,38 @@ def reverse(self) -> None:
 
 
 class JsMap(JsProxy):
```

Comment:
One question: perhaps we should have a `JsMap` and a `JsMutableMap`?

Reply:
I think it is a good idea.